CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ To better understand the changelog, here are some legends we use:
 - 🛠 Refactor
 - 💄 Style
 
+## 1.1.2
+
+`2022-04-04`
```

Comment:
Wrong date

Reply:
yes Great Eyes, working with theo while i make a release is not easy